GNUmakefile

```diff
@@ -93,6 +93,7 @@ cleango: ## Clean up Go cache
 	go clean -modcache -testcache -cache ; \
 
 clean: cleango build tools ## Clean up Go cache and re-install tools
+	go mod tidy
```

Comment:
Should this be `$(GO_VER) mod tidy`?
May need to clean up that `go clean` above also.